newsroom/templates/design_wire.html

```diff
@@ -21,63 +21,7 @@
<i class="icon--search icon--gray"></i></span><div class="search__form input-group"><form class="d-flex align-items-center" role="search" aria-label="search"><input type="text" name="q" class="search__input form-control" placeholder="Search for..." aria-label="Search for..." value=""><div class="search__form__buttons"><button class="btn search__clear" aria-label="Search clear" type="reset"><img src="/static/search_clear.png" width="16" height="16"></button><button class="btn btn-outline-secondary" type="submit">Search</button></div></form></div></div><div class="content-bar__right"><div class="d-flex align-items-center px-2 px-sm-3"><div class="d-flex align-items-center"><label for="all-versions" class="me-2">All Versions</label><div class="react-toggle toggle-background"><div class="react-toggle-track"><div class="react-toggle-track-check"></div><div class="react-toggle-track-x"></div></div><div class="react-toggle-thumb"></div><input id="all-versions" class="react-toggle-screenreader-only" type="checkbox"></div></div></div><div class="d-flex align-items-center px-2 px-sm-3"><div class="d-flex align-items-center flex-column-reverse flex-md-row"><label for="news-only" class="label__news-only me-2">News only</label><div class="react-toggle toggle-background"><div class="react-toggle-track"><div class="react-toggle-track-check"></div><div class="react-toggle-track-x"></div></div><div class="react-toggle-thumb"></div><input id="news-only" class="react-toggle-screenreader-only" type="checkbox"></div></div></div><div class="btn-group list-view__options" data-bs-original-title="" title=""><button class="content-bar__menu" title="Change view" aria-label="Change view" role="button"><i class="icon--list-view"></i></button></div></div><div class="content-bar__right--mobile"><div class="btn-group"><button id="listSearchOptionsButton" class="content-bar__menu" data-bs-toggle="dropdown" aria-haspopup="menu" aria-expanded="false" aria-label="Search Options" role="button"><i class="icon--settings"></i></button><div class="dropdown-menu dropdown-menu-right" aria-labelledby="listSearchOptionsButton"><h6 class="dropdown-header">Search Options</h6><form class="px-3 pt-3"><div class="form-group d-flex align-items-center"><div class="react-toggle toggle-background"><div class="react-toggle-track"><div class="react-toggle-track-check"></div><div class="react-toggle-track-x"></div></div><div class="react-toggle-thumb"></div><input class="react-toggle-screenreader-only" type="checkbox"></div><label for="all-versions" class="mb-0 ms-2">All Versions</label></div><div class="form-group d-flex align-items-center"><div class="react-toggle toggle-background"><div class="react-toggle-track"><div class="react-toggle-track-check"></div><div class="react-toggle-track-x"></div></div><div class="react-toggle-thumb"></div><input class="react-toggle-screenreader-only" type="checkbox"></div><label for="news-only" class="mb-0 ms-2">News Only</label></div></form></div></div></div></nav>
```

Comment:
Not sure what happened here with this code block being all on one line. Could get this separated back out onto multiple lines, so it is easier to understand the structure/layout of these view elements.